packages/lib/src/vis/heatmap/utils.ts

```diff
@@ -40,25 +25,14 @@ export const GRADIENT_RANGE = range(
   GRADIENT_PRECISION
 );
 
-export const TEXTURE_TYPE_BY_DTYPE: Record<
-  DataType<TextureTypedArray>,
-  TextureDataType
-> = {
-  int8: ByteType,
-  int16: ShortType,
-  int32: IntType,
-  uint8: UnsignedByteType,
-  uint16: UnsignedShortType,
-  uint32: UnsignedIntType,
-  float32: FloatType,
```

Comment:
We no longer need this. It's either `FloatType` or `HalfFloatType` (see further down).